CRITICAL - 2022-04-03 10:07:26 --> Invalid file: Auth/singin.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Auth/singin.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('Auth/singin', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(41): view('Auth/singin')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->new()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 10:10:47 --> Undefined property: stdClass::$email
#0 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 55)
#1 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#2 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-03 10:10:47 --> Undefined property: stdClass::$email
#0 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 55)
#1 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#2 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-03 10:15:18 --> Undefined property: stdClass::$email
#0 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 55)
#1 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#2 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-03 10:16:52 --> Undefined property: stdClass::$email
#0 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 55)
#1 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#2 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-03 10:19:14 --> Undefined property: stdClass::$email
#0 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 55)
#1 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#2 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-03 10:33:15 --> Encrypter needs a starter key.
#0 C:\xampp\htdocs\c2sistemaventas\system\Encryption\Encryption.php(124): CodeIgniter\Encryption\Exceptions\EncryptionException::forNeedsStarterKey()
#1 C:\xampp\htdocs\c2sistemaventas\system\Config\Services.php(215): CodeIgniter\Encryption\Encryption->initialize(Object(Config\Encryption))
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(274): CodeIgniter\Config\Services::encrypter()
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(236): Fluent\Auth\Adapters\SessionAdapter->encrypter()
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(133): Fluent\Auth\Adapters\SessionAdapter->recallerCookie(Object(App\Entities\User))
#5 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(104): Fluent\Auth\Adapters\SessionAdapter->login(Object(App\Entities\User), true)
#6 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\Adapters\SessionAdapter->attempt(Array, true)
#7 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->__call('attempt', Array)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(73): Fluent\Auth\Facades\Auth::__callStatic('attempt', Array)
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 11:24:30 --> Encrypter needs a starter key.
#0 C:\xampp\htdocs\c2sistemaventas\system\Encryption\Encryption.php(124): CodeIgniter\Encryption\Exceptions\EncryptionException::forNeedsStarterKey()
#1 C:\xampp\htdocs\c2sistemaventas\system\Config\Services.php(215): CodeIgniter\Encryption\Encryption->initialize(Object(Config\Encryption))
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(274): CodeIgniter\Config\Services::encrypter()
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(236): Fluent\Auth\Adapters\SessionAdapter->encrypter()
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(133): Fluent\Auth\Adapters\SessionAdapter->recallerCookie(Object(App\Entities\User))
#5 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(104): Fluent\Auth\Adapters\SessionAdapter->login(Object(App\Entities\User), true)
#6 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\Adapters\SessionAdapter->attempt(Array, true)
#7 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->__call('attempt', Array)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(73): Fluent\Auth\Facades\Auth::__callStatic('attempt', Array)
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 11:27:16 --> Invalid file: Auth/singin.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Auth/singin.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('Auth/singin', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(41): view('Auth/singin')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->new()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 11:27:42 --> Encrypter needs a starter key.
#0 C:\xampp\htdocs\c2sistemaventas\system\Encryption\Encryption.php(124): CodeIgniter\Encryption\Exceptions\EncryptionException::forNeedsStarterKey()
#1 C:\xampp\htdocs\c2sistemaventas\system\Config\Services.php(215): CodeIgniter\Encryption\Encryption->initialize(Object(Config\Encryption))
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(274): CodeIgniter\Config\Services::encrypter()
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(236): Fluent\Auth\Adapters\SessionAdapter->encrypter()
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(133): Fluent\Auth\Adapters\SessionAdapter->recallerCookie(Object(App\Entities\User))
#5 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(104): Fluent\Auth\Adapters\SessionAdapter->login(Object(App\Entities\User), true)
#6 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\Adapters\SessionAdapter->attempt(Array, true)
#7 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->__call('attempt', Array)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(73): Fluent\Auth\Facades\Auth::__callStatic('attempt', Array)
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 11:30:36 --> Encrypter needs a starter key.
#0 C:\xampp\htdocs\c2sistemaventas\system\Encryption\Encryption.php(124): CodeIgniter\Encryption\Exceptions\EncryptionException::forNeedsStarterKey()
#1 C:\xampp\htdocs\c2sistemaventas\system\Config\Services.php(215): CodeIgniter\Encryption\Encryption->initialize(Object(Config\Encryption))
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(274): CodeIgniter\Config\Services::encrypter()
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(236): Fluent\Auth\Adapters\SessionAdapter->encrypter()
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(133): Fluent\Auth\Adapters\SessionAdapter->recallerCookie(Object(App\Entities\User))
#5 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(104): Fluent\Auth\Adapters\SessionAdapter->login(Object(App\Entities\User), true)
#6 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\Adapters\SessionAdapter->attempt(Array, true)
#7 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->__call('attempt', Array)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(73): Fluent\Auth\Facades\Auth::__callStatic('attempt', Array)
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 11:33:13 --> Encrypter needs a starter key.
#0 C:\xampp\htdocs\c2sistemaventas\system\Encryption\Encryption.php(124): CodeIgniter\Encryption\Exceptions\EncryptionException::forNeedsStarterKey()
#1 C:\xampp\htdocs\c2sistemaventas\system\Config\Services.php(215): CodeIgniter\Encryption\Encryption->initialize(Object(Config\Encryption))
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(274): CodeIgniter\Config\Services::encrypter()
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(236): Fluent\Auth\Adapters\SessionAdapter->encrypter()
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(133): Fluent\Auth\Adapters\SessionAdapter->recallerCookie(Object(App\Entities\User))
#5 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(104): Fluent\Auth\Adapters\SessionAdapter->login(Object(App\Entities\User), true)
#6 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\Adapters\SessionAdapter->attempt(Array, true)
#7 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->__call('attempt', Array)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(73): Fluent\Auth\Facades\Auth::__callStatic('attempt', Array)
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 11:39:55 --> Encrypter needs a starter key.
#0 C:\xampp\htdocs\c2sistemaventas\system\Encryption\Encryption.php(124): CodeIgniter\Encryption\Exceptions\EncryptionException::forNeedsStarterKey()
#1 C:\xampp\htdocs\c2sistemaventas\system\Config\Services.php(215): CodeIgniter\Encryption\Encryption->initialize(Object(Config\Encryption))
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(274): CodeIgniter\Config\Services::encrypter()
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(236): Fluent\Auth\Adapters\SessionAdapter->encrypter()
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(133): Fluent\Auth\Adapters\SessionAdapter->recallerCookie(Object(App\Entities\User))
#5 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Adapters\SessionAdapter.php(104): Fluent\Auth\Adapters\SessionAdapter->login(Object(App\Entities\User), true)
#6 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\Adapters\SessionAdapter->attempt(Array, true)
#7 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->__call('attempt', Array)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Auth\AuthenticatedSessionController.php(73): Fluent\Auth\Facades\Auth::__callStatic('attempt', Array)
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Auth\AuthenticatedSessionController->create()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\AuthenticatedSessionController))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 11:53:07 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('logout', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:54:29 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('logout', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:54:33 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('logout', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:56:32 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:56:35 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:56:39 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:56:42 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:56:52 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:56:54 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 11:58:00 --> Unauthenticated.
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(65): Fluent\Auth\Filters\AuthenticationFilter->unauthenticated(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\AuthenticationFilter.php(35): Fluent\Auth\Filters\AuthenticationFilter->authenticate(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#2 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\AuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-04-03 13:03:24 --> Invalid file: jk.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jk.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('jk.php', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(14): view('jk.php')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->dashboard()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 13:12:52 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 13:13:19 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 13:13:59 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 13:14:24 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 16:39:02 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->guard('web')
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\RedirectAuthenticatedFilter.php(22): Fluent\Auth\Facades\Auth::__callStatic('guard', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\RedirectAuthenticatedFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#6 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('register', 'before')
#7 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-04-03 16:39:05 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:42:45 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->guard('web')
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\RedirectAuthenticatedFilter.php(22): Fluent\Auth\Facades\Auth::__callStatic('guard', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\RedirectAuthenticatedFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#6 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('register', 'before')
#7 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-04-03 16:42:54 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:42:55 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:43:53 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:44:57 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:45:07 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:45:59 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:47:12 --> Fluent\Auth\Adapters\SessionAdapter::__construct(): Argument #2 ($provider) must be of type Fluent\Auth\Contracts\UserProviderInterface, App\Models\UserModel given, called in C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php on line 276
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(276): Fluent\Auth\Adapters\SessionAdapter->__construct('web', Object(App\Models\UserModel), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Session\Session))
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:48:41 --> Class "App\Models\UserModel" not found
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(273): Fluent\Auth\AuthManager->createUserProvider('users')
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(339): Fluent\Auth\AuthManager->guard()
#4 C:\xampp\htdocs\c2sistemaventas\app\Views\welcome_message.php(252): Fluent\Auth\AuthManager->__call('check', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#8 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Home.php(9): view('welcome_message')
#9 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Home->index()
#10 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-04-03 16:59:19 --> Class "App\Models\UserModel" not found
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(273): Fluent\Auth\AuthManager->createUserProvider('users')
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->guard('web')
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\RedirectAuthenticatedFilter.php(22): Fluent\Auth\Facades\Auth::__callStatic('guard', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\RedirectAuthenticatedFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#6 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('login', 'before')
#7 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-04-03 17:02:53 --> Class "App\Models\UserModel" not found
#0 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(273): Fluent\Auth\AuthManager->createUserProvider('users')
#1 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(254): Fluent\Auth\AuthManager->createSessionDriver('web', Array)
#2 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\AuthManager.php(79): Fluent\Auth\AuthManager->resolve('web')
#3 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Facades\Auth.php(58): Fluent\Auth\AuthManager->guard('web')
#4 C:\xampp\htdocs\c2sistemaventas\app\ThirdParty\codeigniter4-authentication\src\Filters\RedirectAuthenticatedFilter.php(22): Fluent\Auth\Facades\Auth::__callStatic('guard', Array)
#5 C:\xampp\htdocs\c2sistemaventas\system\Filters\Filters.php(173): Fluent\Auth\Filters\RedirectAuthenticatedFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#6 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(386): CodeIgniter\Filters\Filters->run('login', 'before')
#7 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-04-03 17:04:15 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
ERROR - 2022-04-03 17:04:47 --> Table 'salessystem.user' doesn't exist
CRITICAL - 2022-04-03 17:04:47 --> Call to a member function getResultArray() on bool
#0 C:\xampp\htdocs\c2sistemaventas\app\Controllers\User.php(20): App\Models\UserManagementModel->readUsers()
#1 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\User->index()
#2 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2022-04-03 17:22:26 --> Unknown column 'c.picture' in 'field list'
CRITICAL - 2022-04-03 17:22:26 --> Call to a member function getResultArray() on bool
#0 C:\xampp\htdocs\c2sistemaventas\app\Controllers\User.php(20): App\Models\UserManagementModel->readUsers()
#1 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\User->index()
#2 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-03 18:39:58 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 20:00:42 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-04-03 20:00:44 --> Invalid file: js/main.php
#0 C:\xampp\htdocs\c2sistemaventas\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('js/main.php')
#1 C:\xampp\htdocs\c2sistemaventas\system\Common.php(1089): CodeIgniter\View\View->render('js/main', Array, true)
#2 C:\xampp\htdocs\c2sistemaventas\app\Controllers\Dashboard.php(23): view('js/main')
#3 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(825): App\Controllers\Dashboard->index()
#4 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#5 C:\xampp\htdocs\c2sistemaventas\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\c2sistemaventas\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\c2sistemaventas\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
ERROR - 2022-04-03 20:04:54 --> Cannot delete or update a parent row: a foreign key constraint fails (`salessystem`.`purchase`, CONSTRAINT `purchase_ibfk_3` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`))
